linera-views/src/rocks_db.rs

```diff
@@ -69,6 +69,11 @@ impl KeyValueStore for RocksDbStoreInternal {
         Ok(tokio::task::spawn_blocking(move || client.db.get(&key)).await??)
     }
 
+    async fn test_existence_value(&self, key: &[u8]) -> Result<bool, RocksDbContextError> {
+        let value = self.read_value_bytes(key).await?;
```

Comment:
Should we try to use [`key_may_exist`](https://docs.rs/rocks/latest/rocks/db/struct.DB.html#method.key_may_exist) ?

Reply:
It seems to work.